Documentation/specs/runtime-configuration-file.md

```diff
@@ -282,7 +275,58 @@ Some of the sections in the `.deps.json` file contain data used for runtime comp
 
 An application can be deployed in a "portable" deployment model. In this case, the RID-specific assets of packages are published within a folder structure that preserves the RID metadata. However, `corehost` does not use this folder structure, rather it reads data from the `.deps.json` file. Also, during deployment, the `.exe` file (`corehost` renamed) is not deployed.
 
-In a `portable` target, the package entries may have an additional `subtargets` section detailing RID-specific assets. The `corehost` application should use this data, along with the current RID and the fallback data defined in the `runtimes` section to select one **and only one** "subtarget" out of each package individually. The most specific subtarget should always be selected. In practice, this means selecting the first RID shown on the appropriate line in `runtimes`. For example, given a package containing the following subtargets:
+In the portable deployment model, the `*.runtimeConfig.json` file will contain the `targetRuntime` section:
+
+```json
+{
+    "targetRuntime": {
```

Comment:
Yes, we moved the setting to `runtimeOptions.framework` because of exactly that confusion. I'm updating everything now, but the example at the very top is up-to-date.